hubblestack/defaults/exitcodes.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we only use a couple of these (I've only seen EX_GENERIC) but I'm all for keeping them to use in the future

Reply:
I can't recall... I discussed removing the exit codes with someone. I can't recall if we decided to remove it or not or who I discussed it with, but I definitely decided to keep this -- perhaps vacillating on a previous decision.

I'm not totally sure I understand your comment either, but I think it may be in reference to deciding to remove this file and then deciding to keep it.